app/controllers/v1/client/cart_items_controller.rb

```diff
@@ -0,0 +1,66 @@
+class V1::Client::CartItemsController < ApplicationController
+  before_action :set_cart_item, only: %i[ show edit update  ]
+
+  # GET /cart_items or /cart_items.json
+  def index
+    @cart_items = CartItem.all
+  end
+
+  # GET /cart_items/1 or /cart_items/1.json
+  def show
+  end
```

Comment:
Do you use this code?

Reply:
no this code isn't being used. There is quite of extra code as i used scaffold because of time constraint.